qa/integration-tests/src/test/java/io/camunda/it/tasklist/compatibility/CompatibilityTasklistUserTaskAuthorizationIT.java

```diff
@@ -98,35 +102,16 @@ public class CompatibilityTasklistAssignUserTaskAuthorizationIT {
                   PermissionTypeEnum.UPDATE_USER_TASK,
                   List.of(PROCESS_ID_WITH_JOB_BASED_USERTASK))));
 
-  private static long userTaskKey;
-  private static long userTaskKeyWithJobBasedUserTask;
-  private static long processInstanceKeyWithJobBasedUserTask;
-  private static long anotherUserTaskKeyWithJobBasedUserTask;
-
   @BeforeAll
   public static void beforeAll(@Authenticated(ADMIN_USER_NAME) final CamundaClient adminClient) {
     tasklistRestClient = STANDALONE_CAMUNDA.newTasklistClient();
 
     // deploy a process as admin user
     deployResource(adminClient, "process/process_public_start.bpmn");
+    deployResource(adminClient, "process/process_with_assigned_user_task.bpmn");
     // deploy process with a job based user task process
     deployResource(adminClient, "process/process_job_based_user_task.bpmn");
-
-    // create a process instance
-    final var processInstanceKey = createProcessInstance(adminClient, PROCESS_ID);
-    userTaskKey = awaitUserTaskBeingAvailable(adminClient, processInstanceKey);
-
-    // create a process instance with job based user task
-    processInstanceKeyWithJobBasedUserTask =
-        createProcessInstance(adminClient, PROCESS_ID_WITH_JOB_BASED_USERTASK);
-    userTaskKeyWithJobBasedUserTask =
-        awaitJobBasedUserTaskBeingAvailable(processInstanceKeyWithJobBasedUserTask);
-
-    // create a process instance with job based user task
-    final long anotherProcessInstanceKeyWithJobBasedUserTask =
-        createProcessInstance(adminClient, PROCESS_ID_WITH_JOB_BASED_USERTASK);
-    anotherUserTaskKeyWithJobBasedUserTask =
-        awaitJobBasedUserTaskBeingAvailable(anotherProcessInstanceKeyWithJobBasedUserTask);
```

Comment:
ℹ️ This setup has been moved to separate test methods, for better isolation. As we often need instances for each test